openwisp_monitoring/monitoring/configuration.py

```diff
@@ -0,0 +1,519 @@
+                    'fixed_value': 100,
+                },
+                'query': chart_query['uptime'],
+            },
+            'packet_loss': {
+                'type': 'bar',
+                'title': _('Packet loss'),
+                'description': _(
+                    'Indicates the percentage of lost packets observed in ICMP probes. '
+                    'Obtained with the fping linux program.'
+                ),
+                'summary_labels': [_('Average packet loss')],
+                'unit': '%',
+                'colors': [DEFAULT_COLORS[3]],
+                'order': 210,
+                'query': chart_query['packet_loss'],
+            },
+            'rtt': {
+                'type': 'scatter',
+                'title': _('Round Trip Time'),
+                'description': _(
+                    'Round trip time observed in ICMP probes, measuered in milliseconds.'
+                ),
+                'summary_labels': [
+                    _('Average RTT'),
+                    _('Average Max RTT'),
+                    _('Average Min RTT'),
+                ],
+                'unit': f' {_("ms")}',
+                'order': 220,
+                'query': chart_query['rtt'],
+            },
+        },
+        'alert_settings': {'operator': '<', 'threshold': 1, 'tolerance': 0},
+        'notification': {
+            'problem': {
+                'name': 'Ping PROBLEM',
+                'verb': 'cannot be reached anymore',
+                'level': 'warning',
+                'email_subject': _(
+                    '[{site.name}] {notification.target} is not reachable'
+                ),
+                'message': _(
+                    'The device [{notification.target}]({notification.target_link}) '
+                    '{notification.verb} anymore by our ping messages.'
+                ),
+            },
+            'recovery': {
+                'name': 'Ping RECOVERY',
+                'verb': 'has become reachable',
+                'level': 'info',
+                'email_subject': _(
+                    '[{site.name}] {notification.target} is reachable again'
+                ),
+                'message': _(
+                    'The device [{notification.target}]({notification.target_link}) '
+                    '{notification.verb} again by our ping messages.'
+                ),
+            },
+        },
+    },
+    'config_applied': {
+        'label': _('Configuration Applied'),
+        'name': 'Configuration Applied',
+        'key': 'config_applied',
+        'field_name': 'config_applied',
+        'alert_settings': {'operator': '<', 'threshold': 1, 'tolerance': 5},
+        'notification': {
+            'problem': {
+                'name': 'Configuration Applied PROBLEM',
+                'verb': 'is not applied',
+                'level': 'warning',
+                'email_subject': _(
+                    '[{site.name}] {notification.target} has problem in configuration'
```

Comment:
missing PROBLEM

it should say something like: 

PROBLEM device X configuration status issue